docs/APISocketOptions.md

```diff
@@ -0,0 +1,1511 @@
+[`SRTO_RCVSYN`](#SRTO_RCVSYN) for details), such that when this time is exceeded, 
+it will behave as if in "non-blocking mode". The -1 value means no time limit.
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_RENDEZVOUS
+
+| OptName           | Since | Binding | Type       |  Units  |   Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | ---------- | ------- | ---------- | ------ | --- | ------ |
+| `SRTO_RENDEZVOUS` |       | pre     | `bool`     |         | false      |        | RW  | S      |
+
+- Use Rendezvous connection mode (both sides must set this and both must use the
+procedure of `srt_bind` and then `srt_connect` (or `srt_rendezvous`) to one another.
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_RETRANSMITALGO
+
+| OptName               | Since | Binding | Type   | Units  | Default | Range  | Dir | Entity |
+| --------------------- | ----- | ------- | ------ | ------ | ------- | ------ | --- | ------ |
+| `SRTO_RETRANSMITALGO` | 1.5.0 | pre     | `int`  |        | 0       | [0, 1] | W   | GSD    |
+
+- Retransmission algorithm to use (SENDER option):
+   - 0 - Default (retransmit on every loss report).
+   - 1 - Reduced retransmissions (not more often than once per RTT); reduced 
+     bandwidth consumption.
+
+- This option is effective only on the sending side. It influences the decision 
+as to whether particular reported lost packets should be retransmitted at a 
+certain time or not.
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_REUSEADDR
+
+| OptName           | Since | Binding | Type       |  Units  |   Default  | Range  | Dir | Entity |
+| ----------------- | ----- | ------- | ---------- | ------- | ---------- | ------ | --- | ------ |
+| `SRTO_REUSEADDR`  |       | pre     | `bool`     |         | true       |        | RW  | GSD    |
+
+- When true, allows the SRT socket to use the binding address used already by 
+another SRT socket in the same application. Note that SRT socket uses an 
+intermediate object called Multiplexer to access the underlying UDP sockets, 
+so multiple SRT sockets may share one UDP socket, and the packets received by this 
+UDP socket will be correctly dispatched to the SRT socket to which they are 
+currently destined. This has some similarities to the `SO_REUSEADDR` system socket 
+option, although it's only used inside SRT. 
+
+- *TODO: This option weirdly only allows the socket used in **bind()** to use the 
+local address that another socket is already using, but not to disallow another 
+socket in the same application to use the binding address that the current 
+socket is already using. What it actually changes is that when given an address in 
+**bind()** is already used by another socket, this option will make the binding 
+fail instead of adding the socket to the shared group of that socket that 
+already has bound this address - but it will not disallow another socket to reuse 
+its address.* 
```

Comment:
@ethouris Could you comment on this TODO? Is it still valid?

Reply:
Tell you what. Remove this, and I'll make a PR to fix it. This maybe isn't as bad as described here, but I still think there's some weird inconsistency.